### ISSUE TITLE
Support HealthDCAT-AP for EHDS

### ISSUE THREAD

**Issue by coolharsh55** [w3c/dpv#201]
> tldr; the draft HealthDCAT-AP, an extension of DCAT-AP used in EU open data portals, has recommended use of DPV for modelling legal basis, purposes, and personal data categories. I propose we support this by providing an EHDS extension and work alongside the group to further support use of DPV.

## HealthDCAT-AP

The HealthData@EU project is a pilot version for implementing the European Health Data Space (EHDS) regulation which will enable secondary use of health data from EU organisations. The project is developing a specification called HealthDCAT-AP which extends the DCAT-AP specification currently used to register and manage datasets in the EU (and other national) open data portal. This means all health datasets being shared as part of the EHDS implementation will be required to utilise this specification in order to be accepted and reused through the health data portals.

The draft HealthDCAT-AP specification https://healthdcat-ap.github.io/ shows the current progress regarding these activity, and references DPV explicitly as part of its best practice recommendation. 

Section 1.3 on use of DCAT-AP for secondary health data management states: "To effectively extend DCAT-AP, several best practices are recommended: ... Prioritizing vocabularies used in DCAT-AP for metadata property selection, ... as well as considering Data Privacy Vocabulary (DPV)"

Section 7.6 on the metadata properties for Datasets describes the use of dpv:hasLegalBasis, dpv:hasPersonalData, and dpv:hasPurpose.

Section 8.13 Legal Basis further states the use of dpv:hasJurisdiction and dpv:hasLaw.

Section 11.4.2 Personal Electronic Health Data states " Data holders are also further encouraged to detail the sensitive nature of the dataset:      This is achieved by utilising the personal data property and providing the list of key elements that represent an individual in the dataset. It is required to adopt the predefined classes listed in the extension of the Data Privacy Vocabulary (DPV) Specification "Extended Personal Data categories for DPV" (DPV-PD). The DPV-PD provides additional concepts regarding Personal Data categories. "

Section 12 contains RDF examples showing the use of DPV, though these only use DPV's properties and do not use DPV's taxonomy e.g. purpose is a string/text description and doesn't include DPV's purpose concepts.

## Proposal to support EHDS in DPV

Based on the above, it is clear that the DPV is desired and appreciated in the creation of specifications which must incorporate legally relevant metadata. Achieving this would be a fantastic impact given the scope and scale of the EHDS, and would also be instrumental in paving the way for similar reuses of DPV in other data exchange portals and specifications - especially those within the EU Data Spaces regime which will also be likely to use and extend the DCAT-AP.

To support this, I propose we explicitly support the EHDS implementation, in particular the HealthDCAT-AP, by creating an EU-EHDS extension that provides relevant concepts for legal basis, personal data categories, and purposes as defined in the regulation. Doing this would further justify the role and usefulness of DPV within the HealthDCAT-aP specification, and would show how powerful (and simple to implement) the DPV's modularity and extensibility are in practice.

I also propose working closely with the project to provide best practices and suggestions regarding the use of DPV's taxonomies within the HealthDCAT-AP as a suggestion so as to provide richer metadata and support data governance and regulatory compliance practices. For example, by tagging whether a purpose is specific diagnostic research or population health research, the appropriate approval mechanisms and compliance requirements can be derived and assessed based on the available metadata, as well as used in risk and impact assessments..

**Comment by coolharsh55** [comment]
We now have a EHDS extension that will be shipped as draft in v2.1 - see https://dev.dpvcg.org/legal/eu/ehds. Next task is to try to liase with HealthDCAT-AP and get DPV to provide supporting concepts by modelling EHDS. Moving this issue to v2.2 to reflect the target of providing these for the next release.

**Comment by beanformer** [comment]
HealthDCAT-AP currently (Release 6) uses 3 DPV properties:

1. dpv:hasLegalBasis -> dpv:LegalBasis [0..*]
2. dpv:hasPersonalData -> dpv:PersonalData [0..*]
3. dpv:hasPurpose -> eli:LegalResource [0..*]

It would be nice to develop examples and best practices, especially for 3.

**Comment by coolharsh55** [comment]
Hi @beanformer ; thanks - we're discussing HealthDCAT-AP and EHDS now (#238). We'll work on guidelines, in the meantime, please see:

- [legal bases module](https://w3id.org/dpv/modules/legal_basis)
- [personal data module](https://w3id.org/dpv/modules/personal_data) and [extension](https://w3id.org/dpv/2.3/pd/)
- [purpose module](https://w3id.org/dpv/modules/purposes)

In particular, we want to clarify the use of `dpv:hasPurpose` and recommend using one of the DPV's Purpose taxonomies to benefit from the use of semantics (and our work). `eli:LegalResource` is typically used to refer to published materials, in this case it is likely referring to a legal obligation or a specific aspect of EHDS. For help with these, we're adding concepts to the [EHDS](https://w3id.org/dpv/2.3/legal/eu/ehds/). We're open (and appreciative) to having questions/requirements/etc. to guide this work.

**Comment by beanformer** [comment]
> In particular, we want to clarify the use of `dpv:hasPurpose` and recommend using one of the DPV's Purpose taxonomies to benefit from the use of semantics (and our work). `eli:LegalResource` is typically used to refer to published materials, in this case it is likely referring to a legal obligation or a specific aspect of EHDS. For help with these, we're adding concepts to the [EHDS](https://w3id.org/dpv/2.3/legal/eu/ehds/). We're open (and appreciative) to having questions/requirements/etc. to guide this work.

Additionally, the example [Purpose (dpv:hasPurpose)](https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-6/index.html#dpvhasPurpose) uses dpv:Purpose as Range and the specification for the property in [Dataset](https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-6/index.html#Dataset) says "A free text statement of the purpose of the processing of data or personal data."

**Comment by coolharsh55** [comment]
> Additionally, the example [Purpose (dpv:hasPurpose)](https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-6/index.html#dpvhasPurpose) uses dpv:Purpose as Range and the specification for the property in [Dataset](https://healthdataeu.pages.code.europa.eu/healthdcat-ap/releases/release-6/index.html#Dataset) says "A free text statement of the purpose of the processing of data or personal data."

Yes, precisely to address this. From the example on the HealthDCAT-AP page, the free-text statement is present, and my suggestion is to indicate what's the category of the purpose. As this benefits evaluating whether two purposes are the same or compatible etc.

```turtle
<https://fair.healthdata.be/dataset/d43a158e-7d13-4660-bbc3-9d3f8d5501e5>
    a dcat:Dataset ;
    dpv:hasPurpose [
        a dpv:Purpose;
        skos:broader dpv:ConceptA ; # <--- add
        dct:description "Purpose 1"	
    ] .
<https://fair.healthdata.be/dataset/another-dataset>
    dpv:hasPurpose [
        a dpv:Purpose;
        skos:broader dpv:ConceptB ; # <--- add
        dct:description "Purpose 2"	
    ] .
```

Using above, we can check if "Purpose 1" and "Purpose 2" are compatible, related, etc. by using the hierarchy of concepts in DPV's taxonomy. This is the intended motivation of using semantic (web), and also suggested / best practice.

**Comment by besteves4** [comment]
Some papers on HealthDCAT-AP that we should consider:

- https://ceur-ws.org/Vol-4064/NXDG25-paper5.pdf
- https://ceur-ws.org/Vol-4064/NXDG25-paper3.pdf

**Comment by beanformer** [comment]
> Some papers on HealthDCAT-AP that we should consider:
> 
> * https://ceur-ws.org/Vol-4064/NXDG25-paper5.pdf
> * https://ceur-ws.org/Vol-4064/NXDG25-paper3.pdf

I already read both of them. With regards, to DPV, there seems to be not much influence. They refer to the concepts of DPV Personal Data Vocabulary.

**Comment by IDmachines** [comment]
has HIPAA been taken into consideration or is this out of scope. [https://www.hhs.gov/hipaa/for-professionals/privacy/laws-regulations/index.html](url)

**Comment by beanformer** [comment]
For HealthDCAT-AP, I see no mentioning of HIPAA. For the EHDS, there are guidelines in preparation like [7. DRAFT GUIDELINE FOR HEALTH DATA ACCESS BODIES ON DATA MINIMISATION, PSEUDONYMISATION, ANONYMISATION AND SYNTHETIC DATA](https://tehdas.eu/public-consultations/), which will lead to EU Implementing Acts in March 2027. Also, there has been a major decision of an EU court about pseudonymisation (saying that the reidentification risk must be estimated relative to possible knowledge of the data recipient, not a absolute risk).

**Comment by IDmachines** [comment]
Does that imply that applying the DPV to HIPAA would be its own "profile"?

**Comment by beanformer** [comment]
I'm quite new to DPV. I would say we need an extension for HIPAA like for the [GDNG law](https://w3c.github.io/dpv/2.3/legal/de/gdng/) and refer to the [Personal Data Module](https://w3c.github.io/dpv/2.3/dpv/modules/personal_data.html) and [Personal Data Categories](https://w3c.github.io/dpv/2.3/pd/).
